nextflow_schema.json

```diff
@@ -565,6 +567,22 @@
                     "hidden": true,
                     "fa_icon": "fas fa-question-circle"
                 },
+                "publish_dir_mode": {
+                    "type": "string",
+                    "default": "copy",
+                    "description": "Method used to save pipeline results to output directory.",
+                    "help_text": "The Nextflow `publishDir` option specifies which intermediate files should be saved to the output directory. This option tells the pipeline what method should be used to move these files. See [Nextflow docs](https://www.nextflow.io/docs/latest/process.html#publishdir) for details.",
```

Comment:
Could the description or help text here explicitly refer to the output directory `--outdir` parameter?

Reply:
👍🏽 Fixed in https://github.com/nf-core/viralrecon/pull/285/commits/96d87fc2f7e3523c661da3b368b0db051f123b18